src/components/30-organisms/table-sortable/README.md

```diff
@@ -23,27 +129,29 @@ const model = {
   ],
   tbody: [
     [
-      { html: '<span>11 Gaa</span>' },
+      { html: '<span>11 Test</span>' },
       { html: '<span>Hello</span>' },
       { html: '<span>Cell 2</span>' },
       { html: 'A' },
     ],
     [
-      { html: '<span>1 Gaa</span>' },
-      { html: '<span>zHello 2</span>' },
+      { html: '<span>1 Test</span>' },
+      { html: '<span>Z Hello 2</span>' },
       { html: '<span>Cell 2</span>' },
       { html: 'B' },
     ],
     [
-      { html: '<span>2 Gaa</span>' },
-      { html: '<span>aHello 3</span>' },
+      { html: '<span>2 Test</span>' },
+      { html: '<span>A Hello 3</span>' },
       { html: '<span>Cell 2</span>' },
       { html: 'C' },
```

Comment:
I agree with what Marc wrote somewhere, that quality test data would be nice, but we can also do that later.